### PR TITLE
Update experimental-exporter.md

### PR DESCRIPTION
**Pull request by dmoore247**
Analyze all of the databricks objects and the exporter, added exhaustive exporter support matrix

(Please make sure I analyzed it all correctly)

### CODE REVIEW

File: docs/guides/experimental-exporter.md [hunk 1]
```diff
@@ -61,3 +61,81 @@ Services are just logical groups of resources used for filtering and organizatio
...
+| [databricks_sql_query](docs/resources/sql_query.md) | No |
+| [databricks_sql_visualization](docs/resources/sql_visualization.md) | No |
+| [databricks_sql_widget](docs/resources/sql_widget.md) | No |
+| [databricks_storage_credential](docs/resources/storage_credential.md) | No |
+| [databricks_table](docs/resources/table.md) | No |
+| [databricks_tables](docs/data-sources/table.md) data | No |
+| [databricks_token](docs/resources/token.md) | No |
+| [databricks_user](docs/resources/user.md) | Yes |
+| [databricks_user_instance_profile](docs/resources/user_instance_profile.md) | No (Depricated) |
+| [databricks_workspace_conf](docs/resources/workspace_conf.md) | No |
```
Comment (alexott) [inline docs/guides/experimental-exporter.md:141]: we expose it: https://github.com/databrickslabs/terraform-provider-databricks/blob/master/exporter/importables.go#L838

File: docs/guides/experimental-exporter.md [hunk 2]
```diff
@@ -61,3 +61,81 @@ Services are just logical groups of resources used for filtering and organizatio
...
+| [databricks_secret_acl](docs/resources/secret_acl.md) | Yes |
+| [databricks_secret_scope](docs/resources/secret_scope.md) | Yes |
+| [databricks_spark_version](docs/data-sources/spark_version.md) data | No |
+| [databricks_sql_dashboard](docs/resources/sql_dashboard.md) | No |
+| [databricks_sql_endpoint](docs/resources/sql_endpoint.md) | No |
+| [databricks_sql_global_config](docs/resources/sql_global_config.md) | No |
+| [databricks_sql_permissions](docs/resources/sql_permissions.md) | No |
+| [databricks_sql_query](docs/resources/sql_query.md) | No |
+| [databricks_sql_visualization](docs/resources/sql_visualization.md) | No |
+| [databricks_sql_widget](docs/resources/sql_widget.md) | No |
```
Comment (alexott) [inline docs/guides/experimental-exporter.md:134]: yes, since 0.5.5

File: docs/guides/experimental-exporter.md [hunk 3]
```diff
@@ -61,3 +61,81 @@ Services are just logical groups of resources used for filtering and organizatio
...
+| [databricks_pipeline](docs/resources/pipeline.md) | No |
+| [databricks_repo](docs/resources/repo.md) | Yes |
+| [databricks_schema](docs/resources/schema.md) | No |
+| [databricks_schemas](docs/data-sources/schema.md) data | No |
+| [databricks_secret](docs/resources/secret.md) | Yes |
+| [databricks_secret_acl](docs/resources/secret_acl.md) | Yes |
+| [databricks_secret_scope](docs/resources/secret_scope.md) | Yes |
+| [databricks_spark_version](docs/data-sources/spark_version.md) data | No |
+| [databricks_sql_dashboard](docs/resources/sql_dashboard.md) | No |
+| [databricks_sql_endpoint](docs/resources/sql_endpoint.md) | No |
```
Comment (alexott) [inline docs/guides/experimental-exporter.md:129]: yes, since 0.5.5

File: docs/guides/experimental-exporter.md [hunk 4]
```diff
@@ -61,3 +61,81 @@ Services are just logical groups of resources used for filtering and organizatio
...
+| [databricks_permissions (mlflow registered model)](docs/resources/permissions.md) | No |
+| [databricks_permissions (sql_endpoint) ](docs/resources/permissions.md) | No |
+| [databricks_pipeline](docs/resources/pipeline.md) | No |
+| [databricks_repo](docs/resources/repo.md) | Yes |
+| [databricks_schema](docs/resources/schema.md) | No |
+| [databricks_schemas](docs/data-sources/schema.md) data | No |
+| [databricks_secret](docs/resources/secret.md) | Yes |
+| [databricks_secret_acl](docs/resources/secret_acl.md) | Yes |
+| [databricks_secret_scope](docs/resources/secret_scope.md) | Yes |
+| [databricks_spark_version](docs/data-sources/spark_version.md) data | No |
```
Comment (alexott) [inline docs/guides/experimental-exporter.md:127]: data sources doesn't make sense to expose. I recommend to leave only resources in this list

File: docs/guides/experimental-exporter.md [hunk 5]
```diff
@@ -61,3 +61,81 @@ Services are just logical groups of resources used for filtering and organizatio
...
+| [databricks_permissions (cluster_policy)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (repo)](docs/resources/permissions.md) | Yes |
+| [databricks_permissions (token)](docs/resources/permissions.md) | No |
+| [databricks_permissions (password)](docs/resources/permissions.md) | No |
+| [databricks_permissions (delta live tables pipeline)](docs/resources/permissions.md) | No |
+| [databricks_permissions (notebook)](docs/resources/permissions.md) | No |
+| [databricks_permissions (directory)](docs/resources/permissions.md) | No |
+| [databricks_permissions (mlflow experiment)](docs/resources/permissions.md) | No |
+| [databricks_permissions (mlflow registered model)](docs/resources/permissions.md) | No |
+| [databricks_permissions (sql_endpoint) ](docs/resources/permissions.md) | No |
```
Comment (alexott) [inline docs/guides/experimental-exporter.md:119]: yes, since 0.5.5

File: docs/guides/experimental-exporter.md [hunk 6]
```diff
@@ -61,3 +61,81 @@ Services are just logical groups of resources used for filtering and organizatio
...
+| [databricks_mlflow_experiment](docs/resources/mlflow_experiment.md) | No |
+| [databricks_mws_credentials](docs/resources/mws_credentials.md) | No |
+| [databricks_mws_customer_managed_keys](docs/resources/mws_customer_managed_keys.md) | No |
+| [databricks_mws_log_delivery](docs/resources/mws_log_delivery.md) | No |
+| [databricks_mws_networks](docs/resources/mws_networks.md) | No |
+| [databricks_mws_storage_configurations](docs/resources/mws_storage_configurations.md) | No |
+| [databricks_mws_workspaces](docs/resources/mws_workspaces.md) | No |
+| [databricks_node_type](docs/data-sources/node_type.md) data | No |
+| [databricks_notebook](docs/resources/notebook.md) | Yes |
+| [databricks_notebook](docs/data-sources/notebook.md) data | Yes |
```
Comment (alexott) [inline docs/guides/experimental-exporter.md:104]: No information is exported for data sources - remove all of them

**Review by alexott — CHANGES_REQUESTED** [review]

**Review by nfx — APPROVED** [review]

File: docs/guides/experimental-exporter.md (hunk 5 above)
Comment (dmoore247) [reply docs/guides/experimental-exporter.md]: @alexott Is this to say all permissions in https://docs.databricks.com/dev-tools/api/latest/permissions.html are exportable?
I'm looking for a way to note what ACLs are not exportable to assist in planning migration projects.

Comment (alexott) [reply docs/guides/experimental-exporter.md]: @dmoore247 all resources that have similar block are exporting permissions: https://github.com/databrickslabs/terraform-provider-databricks/blob/master/exporter/importables.go#L1021

**Pull request merged**